geniza/footnotes/admin.py

```diff
@@ -1,53 +1,81 @@
+from adminsortable2.admin import SortableInlineAdminMixin
 from django.contrib import admin
 from django.contrib.admin import SimpleListFilter
 from django.contrib.contenttypes.admin import GenericTabularInline
-from geniza.footnotes.models import Footnote, Source, SourceType, Creator
+from django.contrib.postgres.aggregates import ArrayAgg
+from django.db import models
+from django.db.models.functions import Concat
 from modeltranslation.admin import TabbedTranslationAdmin
 
+from geniza.footnotes.models import Authorship, Creator, Footnote, Source, \
+    SourceLanguage, SourceType
+
+
+class AuthorshipInline(SortableInlineAdminMixin, admin.TabularInline):
+    model = Authorship
+    autocomplete_fields = ['creator']
+    fields = ('creator', 'sort_order')
+    extra = 1
+
 
 @admin.register(Source)
 class SourceAdmin(TabbedTranslationAdmin, admin.ModelAdmin):
     list_display = (
-        'all_authors', 'title', 
+        'title', 'all_creators',
         'year',
         'edition',
     )
 
     search_fields = (
-        'title', 'all_authors', 'year'
+        'title', 'creators__first_name', 'creators__last_name' 'year'
```

Comment:
is this missing a comma before `'year'`?

Reply:
um yeah, how's that not throwing an error? thanks for catching